react/README.md

```diff
@@ -1453,7 +1453,94 @@ export default class TextInput extends React.Component {
 
 ### Event handling in loops
 
-Coming soon....
+Chances are that if you are passing event handlers to child components created in a loop, you're going to need some way to uniquely identify which child component caused the event handler to occur. Pass additional arguments to [`Function.prototype.bind`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/bind) to include the unique identifier:
+
+```js
+// shared data
+const TEAMS = {
+    'warriors': {
+        name: 'Golden State Warriors',
+        url: 'http://www.nba.com/warriors'
+    },
+    '49ers': {
+        name: 'San Francisco',
+        url: 'http://www.49ers.com'
+    },
+    'raiders': {
+        name: 'Oakland Raiders',
+        url: 'http://www.raiders.com'
+    },
+    'giants': {
+        name: 'San Francisco Giants',
+        url: 'http://sanfrancisco.giants.mlb.com'
+    },
+    'athletics': {
+        name: 'Oakland Athletics',
+        url: 'http://oakland.athletics.mlb.com'
+    },
+    'sharks': {
+        name: 'San Jose Sharks',
+        url: 'http://sharks.nhl.com'
+    }
+};
+
+
+// good
+export default class TeamPicker extends React.Component {
+    _handleTeamClick(teamId) {
+        location.href = TEAMS[teamId].url;
+    }
+
+    render() {
+        let teamButtons = Object.keys(TEAMS).map((teamId) => (
+            <button onClick={this._handleTeamClick.bind(this, teamId)} key={teamId}>
```

Comment:
So, this is a case where the `teamId` is a number, but it's not the `index` in the map, so it's ok to use it as the `key`?